docusaurus.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   title: '<mapml-viewer> documentation',
-  tagline: 'A custom <mapml-viewer> and <layer-> element suite',
+  tagline: 'Embed map viewers into your web pages, using HTML only.',
```

Comment:
Should this rather say <q>using only HTML</q> instead of <q>using HTML only</q>?